src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,28 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\Search\Bridge\Symfony\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+final class Configuration implements ConfigurationInterface
+{
+    /**
+     * Generates the configuration tree builder.
+     *
+     * @return \Symfony\Component\Config\Definition\Builder\TreeBuilder The tree builder
+     */
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('search');
+
+        $treeBuilder->getRootNode()
+            ->children()
+                ->booleanNode('use_commands')->defaultFalse()->end()
+                ->booleanNode('use_listeners')->defaultFalse()->end()
```

Comment:
I see no advantage to defining these configuration options - if you're using the search package both are required to achieve the functionality that this package provides?

Reply:
I agree there is no advantage in the actual usage but during testing, sometimes we don't want the listeners to execute... I think we can leave `use_listeners` but set the default to true? This way, we can enable/disable listeners in the user/app level. What chu think?

Reply:
I'm typically against adding specific functionality to ease testing issues.

Realistically what should be happening here is the listener should be stubbed out for functional tests because we should be providing applications the ability to check the listener fired correctly, not requiring them to have different testing envs.

Reply:
Ah yes. Got it.